### PR TITLE
Clarify terms related to various osu! teams where possible

### PR DESCRIPTION
**Pull request by TicClick**
part of https://github.com/ppy/osu-wiki/issues/1698 (does NOT solve it completely). think this change is only specific to English articles, so no `outdated: true` markers scattered across translations

### CODE REVIEW

File: wiki/How_you_can_help!/en.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Below are just a few of the ways you can contribute:
 
 ## Contribute to the future of osu!
 
-Started in 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which is what you see today! If you would like to help make these projects even better and bring us closer to the future of osu!, there are a number of things you can do, but below are just a few examples:
+Started in 2017, the osu!lazer project is the current focus of the team of developers, along with the new website, which is what you see today! If you would like to help make these projects even better and bring us closer to the future of osu!, there are a number of things you can do, but below are just a few examples:
```
Comment (peppy) [inline wiki/How_you_can_help!/en.md:63]: team of developers reads weird. i'd probably just leave as osu! team or team?

File: wiki/History_of_osu!/2015/en.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ Despite [promises from peppy on his blog](https://blog.ppy.sh/post/132009865043/
...
+After months of work, a full migration from [DirectX](https://en.wikipedia.org/wiki/DirectX "Wikipedia") to [OpenGL](https://en.wikipedia.org/wiki/OpenGL "Wikipedia") graphics rendering was completed and pushed out to all users on the Stable release stream on 16 October 2015. However, users who were not able to run the new update due to compatibility issues were switched to a new release stream named "Stable (Fallback)", which used the old DirectX graphics rendering. It was meant to be continuously supported alongside the Stable release until the team of developers felt confident enough in the compatibility of the OpenGL rendering for all users [[28,29,30,31,32]][r].
```
Comment (peppy) [inline wiki/History_of_osu!/2015/en.md:84]: also reads weird

File: wiki/History_of_osu!/2015/en.md [hunk 3]
```diff
@@ -51,9 +51,9 @@ The ability to see what a replay would look like without [mods](/wiki/Game_modif
...
+On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test cutting-edge features, contribute graphical/sound resources, or get help working with osu! services could come and join the team and talk to the people from different [osu! teams](/wiki/People/The_Team) directly [[21]][r].
 
-*Note: There was still a private Slack team that the osu! team used for internal communications, which was not accessible or viewable to outside users; these two teams were independent of each other.*
+*Note: There was still a private Slack team that the osu! teams used for internal communications, which was not accessible or viewable to outside users; these two teams were independent of each other.*
```
Comment (peppy) [inline wiki/History_of_osu!/2015/en.md:56]: i dunno about plural when the main wiki page is "the team" probably fine to just leave as singular for such general references?

File: wiki/Projects/en.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ outdated: true
 
 ## Community
 
-*Caution: Use at your own risk! The projects listed below are not endorsed by osu! nor have any support from the osu! team.*
+*Caution: Use at your own risk! The projects listed below are not endorsed by osu!, nor they have any official support.*
```
Comment (peppy) [inline wiki/Projects/en.md:37]:
```suggestion
*Caution: Use at your own risk! The projects listed below are not endorsed by osu!, nor do they have any official support.*
```

File: wiki/Rules/Visual_Content_Considerations/en.md [hunk 5]
```diff
@@ -55,4 +55,4 @@ If you are unsure about an image that is going to be included in a beatmap in an
 
 **The result of any vote is final**, unless the image is significantly altered in an attempt to resolve any potentially problematic elements raised during discussion.
 
-At all times, the [osu!team](/wiki/People/The_Team) reserves the right to unilaterally declare any piece of content either valid or invalid at their discretion, regardless of any previous voting held on the matter.
+At all times, the voting teams reserve the right to unilaterally declare any piece of content either valid or invalid at their discretion, regardless of any previous voting held on the matter.
```
Comment (peppy) [inline wiki/Rules/Visual_Content_Considerations/en.md:58]: also reads weird. not sure what a "voting team" is

File: wiki/osu!talk/en.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 ![](img/osu!talk.jpg "osu!talk logo")
 
-**osu!talk** is a podcast series hosted by [ztrot](https://osu.ppy.sh/users/6347) with [deadbeat](https://osu.ppy.sh/users/128370) as the co-host. It focuses on interviewing various users of the osu! community. In addition to this, there may be some group discussions about different things happening around the community, such as changes to the osu! team, or often discussed long lasting issues, like overmapping.
+**osu!talk** is a podcast series hosted by [ztrot](https://osu.ppy.sh/users/6347) with [deadbeat](https://osu.ppy.sh/users/128370) as the co-host. It focuses on interviewing various users of the osu! community. In addition to this, there may be some group discussions about different things happening around the community, such as changes to the [osu! teams](/wiki/People/The_Team), or often discussed long lasting issues, like overmapping.
```
Comment (peppy) [inline wiki/osu!talk/en.md:5]: prolly singular too

**Review by peppy — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/2015/en.md (hunk 3 above)
Comment (TicClick) [reply wiki/History_of_osu!/2015/en.md]: I suppose, although it doesn't solve the issue in general

File: wiki/Rules/Visual_Content_Considerations/en.md (hunk 5 above)
Comment (TicClick) [reply wiki/Rules/Visual_Content_Considerations/en.md]: to be frank, I'm not sure what `the osu!team` is either, considering that not all people from `People/The team`, and not all user groups listed there, have access to voting. in this case, `the osu!team`, or `the osu! team`, becomes `the people and teams who vote, but not necessarily these who are on the page`, which is about as ambiguous as [other usages of it](https://github.com/search?q=osu%21team+repo%3Appy%2Fosu-wiki+filename%3Aen.md&type=Code&l=&l=)

Comment (TicClick) [reply wiki/Rules/Visual_Content_Considerations/en.md]: changed it to "the osu! team" but the issue still stands

**Review by peppy — APPROVED** [review]

**Pull request merged**